tests/Doctrine/Tests/Annotations/AbstractReaderTest.php

```diff
@@ -408,6 +408,26 @@ public function testInvalidAnnotationButIgnored()
         self::assertEmpty($reader->getPropertyAnnotations($class->getProperty('foo')));
     }
 
+    public function testGloballyIgnoredAnnotationNotIgnored()
+    {
+        $reader = $this->getReader();
+        $class  = new \ReflectionClass(Fixtures\ClassDDC1660::class);
+
+        spl_autoload_register($testLoader = function ($className) {
```

Comment:
Let's move the definition of the function outside of `spl_autoload_register()` call, things will be more readable.

Reply:
Also making it static and add type declaration to it:

```diff
- $testLoader = function ($className) {
+ $testLoader = static function (string $className) : bool {
```

Reply:
Again, there's not a single place in the entire code base of the project where `static function`, and so is true for type declarations. I'm not saying they're not cool or something, just are not the purpose of this PR.

Reply:
See https://github.com/doctrine/annotations/pull/235#discussion_r